pkg/config/config.go

```diff
@@ -100,6 +100,7 @@ const (
 	// MaxTokenLimit is the max token limit value.
 	MaxTokenLimit  = 1024 * 1024
 	DefSchemaLease = 45 * time.Second
+	UnavailableIP  = "<nil>"
```

Comment:
why put the constant here?

Reply:
Because the config `AdvertiseAddress` is defined here. `UnavailableIP` is only used by `AdvertiseAddress`.